packages/webdriverio/src/scripts/scrollIntoViewDeltaCalcScript.ts

```diff
@@ -0,0 +1,287 @@
+/**
+ * adopted to w3c delta
+ * original source: https://github.com/scroll-into-view/compute-scroll-into-view
+ * Copyright by https://github.com/stipsan
+ */
```

Comment:
I think if we end up using this, we should add `compute-scroll-into-view` as dependency as we should avoid to have maintain this ourselves

Reply:
It's not possible, the dependency has been rewritten totally and adopted for obtaining delta, i even couldn't have mentioned original. Original library provide coordinates for full scroll path, but in case of actions are necessary only deltaX,Y because actions is moving a scroll to end by default.